manifests/config.pp

```diff
@@ -192,9 +192,9 @@
     }
   }
 
-  case $::osfamily {
+  case $facts['os']['family'] {
     'Debian': {
-      if versioncmp($::operatingsystemmajrelease, '16.04') >= 0 {
+      if versioncmp($facts['os']['release']['major'], '16.04') >= 0 {
```

Comment:
I know this is a 1-to-1 conversion but the major is ```16``` so does that mean it'll only be true for ```17.04```? Maybe the ```['full']``` fact would be better.

Things might also need explicit Debian/Ubuntu checks if you add Debian 9 to the mix but that's for later.

Reply:
maybe we should just do major => 16?
I don't think the module supports 16 yet, though we should add support for it (someone was asking for it this morning)

Reply:
Though I guess it should eval true anyway because it's `versioncmp()`? I'll just switch to full assuming tests pass.

Would be great if someone could update the acceptance tests for 16.x

Reply:
Updated in new version